ci/shellcheck.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -euo
+set -euxo
```

Comment:
isn't `pipefail` needed here too, so it fails if shell is not installed?

Reply:
`sh` does not support this option